app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -0,0 +1,246 @@
+                adj_type = 'FINDDx / MUHC independent evaluation'
+                # Also note these must be divided by 100
+                se = (estimate['ind_se']) / 100
+                sp = (estimate['ind_sp']) / 100
+                se_n = float(estimate['ind_se_n'][0]) * 100 if estimate['ind_se_n'] is not None else 30
+                sp_n = float(estimate['ind_sp_n'][0]) * 100 if estimate['ind_sp_n'] is not None else 80
+
+            # Author evaluation is available
+            elif pd.notna(estimate['se_n']) and pd.notna(estimate['sp_n']) and \
+                    pd.notna(estimate['sensitivity']) and pd.notna(estimate['specificity']):
+                if estimate['test_validation'] and \
+                        'Validated by independent authors/third party/non-developers' in estimate['test_validation']:
+                    adj_type = 'Author-reported independent evaluation'
+                else:
+                    adj_type = 'Test developer / manufacturer evaluation'
+                se = estimate['sensitivity']
+                sp = estimate['specificity']
+                se_n = estimate['se_n'] if pd.notna(estimate['se_n']) else 30
+                sp_n = estimate['sp_n'] if pd.notna(estimate['sp_n']) else 80
+
+            # Manufacturer evaluation is available
+            elif pd.notna(estimate['sensitivity']) and pd.notna(estimate['specificity']):
+                adj_type = 'Test developer / manufacturer evaluation'
+                se = estimate['sensitivity']
+                sp = estimate['specificity']
+                # per FDA minimum requirements https://www.fda.gov/medical-devices/
+                # coronavirus-disease-2019-covid-19-emergency-use-authorizations-medical-devices/
+                # eua-authorized-serology-test-performance
+                se_n, sp_n = 30, 80
+
+            else:
+                # if there is no matched adjusted estimate available:
+                found_test_type = False
+                for test_type in ['LFIA', 'CLIA', 'ELISA']:
+                    if estimate['test_type'] and test_type == estimate['test_type']:
+                        se = bastos_estimates[test_type]['se']['50']
+                        sp = bastos_estimates[test_type]['sp']['50']
+                        se_n = bastos_estimates[test_type]['se']['n']
+                        sp_n = bastos_estimates[test_type]['sp']['n']
+                        adj_type = 'Used Bastos SR/MA data; no sens, spec, or author adjustment available'
+                        found_test_type = True
+                        break
+                if not found_test_type:
+                    adj_type = 'No data altogether'
+
+            if adj_type == 'No data altogether':
+                adj_prev = None
+                se = None
+                sp = None
+                lower = None
+                upper = None
+            else:
+                print(f'ADJUSTING ESTIMATE AT INDEX {estimate.name}', adj_type)
```

Comment:
let's use logger instead of printing here if we want this logged somewhere

Reply:
the point of logging this earlier was to log progress through the lengthy process - not sure that this is the best mechanism to do this anymore, but could be ok - at your discretion